src/header.py

```diff
@@ -0,0 +1,6 @@
+#!/opt/bin/lv_micropython
+import lvgl as lv
+import init_gui
```

Comment:
Where is `init_gui` defined?

Reply:
Added it. It initializes the SDL, ili9341 drivers or the drivers needed for the t-watch